src/Files/ContentFile.ts

```diff
@@ -0,0 +1,19 @@
+import type { BaseServer } from "../Server/BaseServer";
```

Comment:
Why is this in "Files" but the others are in "Paths"? Seems like an artificial distinction.

Reply:
Yeah I was thinking about moving e.g. TextFile, Script, etc into "Files" (right now TextFiles is in root) but ended up not doing that. It is weird with nothing else in there.